api/lib/config.js

```diff
@@ -142,7 +143,7 @@ module.exports = (function() {
     config.domain.pixOrga = 'https://orga.pix';
 
     config.mailing.enabled = false;
-    config.mailing.provider = 'mailjet';
+    config.mailing.provider = 'sendinblue';
```

Comment:
Remarque mineure: pour éviter les magic values (meme chose dans api/lib/infrastructure/mailers/mailer.js ?)
```suggestion
   const MAILER = { SEND_IN_BLUE: 'sendinblue', MAILJET: 'mailjet' }
    config.mailing.provider = MAILER.SEND_IN_BLUE
```

Reply:
J'ai pas compris.
Toutes les valeurs (NODE_ENV = 'test') sont données directement.
J'ai modifié le nom du provider, car maintenant 'mailjet' est obsolete.